AUTHORS

```diff
@@ -2,6 +2,7 @@
 Acid147 <norman.rauschen@web.de>
 Andrew Gerrand <nf@wh3rd.net>
 Ben Echols <lologarithm@gmail.com>
+Beoran <beoran@gmail.com>
```

Comment:
It's OK if they are `.gitconfig`'s name and email :D

Reply:
You mean I don't need to do this? It's automati? If so, I should leave out this change then?

Reply:
Basically I'lld do this change and you don't have to add :-)

Reply:
Ok, So I won't then. :)